multimap/src/main/java/org/infinispan/multimap/impl/SortedSetBucket.java

```diff
@@ -104,7 +103,7 @@ public Collection<ScoredValue<V>> inter(Collection<ScoredValue<V>> inputValues,
                                           AggregateFunction function) {
       if (inputValues == null) {
          return scoredEntries.stream()
-               .map(s -> new ScoredValue<>(s.score * weight, s.wrappedValue()))
+               .map(s -> new ScoredValue<>(s.score() * weight, s.wrappedValue()))
```

Comment:
It seems that if `inputValues` is null `inter` returns the whole `scoredEntries`, while if `inputValues` is an empty collection an empty collection is returned. it this ok?

Reply:
yes, because inputvalues null means first call